clj-refactor.el

```diff
@@ -2247,15 +2247,15 @@ See: https://github.com/clojure-emacs/clj-refactor.el/wiki/cljr-update-project-d
               (insert name)
             (mc/create-fake-cursor-at-point)))
         (re-search-forward "\\[")
-        (when (s-present? locals)
+	(when (not (or (null locals) (string= "" locals)))
```

Comment:
i would possibly rather go for
`string-empty-p` instead of `string= ""` for readability and perhaps factor this pred out in its own function? something like `cljr--string-present-p`?